flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBStateBackend.java

```diff
@@ -166,6 +169,11 @@
 	/** Whether we already lazily initialized our local storage directories. */
 	private transient boolean isInitialized;
 
+	/**
+	 * Max consumed memory size for one batch in {@link RocksDBWriteBatchWrapper}, default value 2mb.
+	 */
+	private long batchSize;
```

Comment:
Suggest to change all "batchSize/BATCH_SIZE" to "writeBatchSize/WRITE_BATCH_SIZE" in this class.

Reply:
+1

Reply:
done